### PR TITLE
[#5042] release translations

### LINKED ISSUE #4096: testing 5024: wording changed date
**URL:** https://meinberlin-dev.liqd.net/projekte/burgerhaushalt-spandau/?mode=list
**user:** any
**expected behaviour:** translation changed date should be "bearbeitet am"
**behaviour:** is "geändert am"
**important screensize:**
**device & browser:** 
**Comment/Question:** 

Screenshot?

### PR DESCRIPTION
**Pull request by sabinammm**
Translation task for US #5042 prepare release.
This also fixes #4096

### CODE REVIEW

File: locale/de_DE/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -746,20 +746,20 @@ msgstr "Keine Ergebnisse"
...
 msgid_plural "you have %s votes left"
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "Sie haben noch %s Stimmen."
+msgstr[1] "Sie haben noch %s Stimmen."
 
 #: meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jsx:12
 msgid "modified on"
-msgstr "geändert am"
+msgstr "bearbeitet am"
```
Comment (fuzzylogic2000) [inline locale/de_DE/LC_MESSAGES/djangojs.po:754]: Which wording do we use in the other (Django) list? I couldn't find "modified on" and I think, we should use the same wording for both lists. I will merge anyway, but could you check that out, please?

**Review by fuzzylogic2000 — APPROVED** [review]
Looks good!

File: locale/de_DE/LC_MESSAGES/djangojs.po (hunk 1 above)
Comment (fuzzylogic2000) [reply locale/de_DE/LC_MESSAGES/djangojs.po:754]: @sabinammm Did you see my comment? Did you find out about the other wording?

File: locale/de_DE/LC_MESSAGES/djangojs.po [hunk 2]
```diff
@@ -1282,3 +1195,56 @@ msgid "More information can be found in the privacy policy of Vimeo under: "
 msgstr ""
 "Weitere Informationen finden Sie in der Datenschutzerklärung von Vimeo "
 "unter: "
+
```
Comment (phillimorland) [inline locale/de_DE/LC_MESSAGES/djangojs.po:1198]: @fuzzylogic2000 this is all the deletd ones :(

Comment (fuzzylogic2000) [reply locale/de_DE/LC_MESSAGES/djangojs.po:1198]: @phillimorland Yes, and there are even more in another commit. But @sabinammm is on it and will put them all back!

Comment (sabinammm) [reply locale/de_DE/LC_MESSAGES/djangojs.po:1198]: @fuzzylogic2000 actually I checked translation PRs through November 2021 and this is the only commit with automatically removed translations that I am re-adding. Is there another one that I missed?

Comment (fuzzylogic2000) [reply locale/de_DE/LC_MESSAGES/djangojs.po:1198]: @sabinammm In the other commit/PR, I posted on taiga, we did not loose any translations, but only changed all the strings that were added to the old ones? 
commit https://github.com/liqd/a4-meinberlin/commit/4ab744ae8d1b52667964bf3b078fd2c6862ed441#diff-2833e2202830b2667d27de3334567174802d8e78f0ce316657b743fdcf47d038
from https://github.com/liqd/a4-meinberlin/pull/4055

Comment (sabinammm) [reply locale/de_DE/LC_MESSAGES/djangojs.po:1198]: @fuzzylogic2000 Oh, thanks for the repost :) The strings that were removed in that [commit](https://github.com/liqd/a4-meinberlin/commit/4ab744ae8d1b52667964bf3b078fd2c6862ed441#diff-2833e2202830b2667d27de3334567174802d8e78f0ce316657b743fdcf47d038), are now coming from a4 and are all there in the current translation files. These are also all in js-files. I only found 1 string that was removed in the other commit that comes from an a4-jsx-file. So I think we are good to go at this time, right?

**Pull request merged**